pkgs/by-name/re/refine/package.nix

```
@@ -0,0 +1,102 @@
+{
+  lib,
+  fetchFromGitLab,
+  nix-update-script,
+  blueprint-compiler,
+  desktop-file-utils,
+  libadwaita,
+  meson,
+  ninja,
+  pkg-config,
+  wrapGAppsHook4,
+  python3,
+  libxml2,
+  python3Packages,
+  libportal,
+  libportal-gtk4,
+  appstream,
+  gtk4,
+  glib,
+}:
+
+let
+  libadwaita' = libadwaita.overrideAttrs (
+    finalAttrs: oldAttrs: {
```

```suggestion
    oldAttrs: {
```
Nitpick, but realized we don't this since we aren't re-using any variables we have overriden